includes/class-llms-media-protector.php

```diff
@@ -0,0 +1,754 @@
+<?php
+/**
+ * LLMS_Media_Protector class
```

Comment:
```suggestion
 * LLMS_Media_Protector class.
```

Reply:
The [WordPress](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#6-file-headers) and [LifterLMS](https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#file-headers-1) coding standard document say that the summary line in **file** level DocBlocks should not end with a period.
I don't understand why, because it is inconsistent.

Reply:
Jeez you're right, ok then, sorry.